ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-ospf.j2

```diff
@@ -120,6 +120,44 @@
 {%             endif %}
 {%         endfor %}
 {%     endif %}
+{# OSPF Timers #}
+{%     set has.found = false %}
+{%     for process in router_ospf.process_ids %}
+{%         if router_ospf.process_ids[process].timers is arista.avd.defined %}
+{%             set has.found = true %}
+{%         endif %}
+{%     endfor %}
+{%     if has.found is arista.avd.defined(true) %}
+
+### Router OSPF timers
+
+| Process ID | LSA rx | LSA tx | SPF |
+| ---------- | ------ | ------ | --- |
+{%         for process in router_ospf.process_ids | arista.avd.natural_sort %}
+{%             if router_ospf.process_ids[process].timers is arista.avd.defined %}
+{%                 if router_ospf.process_ids[process].timers.lsa.rx is defined %}
+{%                     set lsa_rx = router_ospf.process_ids[process].timers.lsa.rx %}
+{%                 else %}
+{%                     set lsa_rx = 'unset' %}
+{%                 endif %}
+{%                 if router_ospf.process_ids[process].timers.lsa.tx is defined %}
```

Comment:
this should test for all the child values with `and`. Also use `arista.avd.defined`

Reply:
Fixed with e4265b9a